test/fixtures/index.js

```diff
@@ -57,6 +57,7 @@ module.exports = {
 
   inputJpg: getPath('2569067123_aca715a2ee_o.jpg'), // http://www.flickr.com/photos/grizdave/2569067123/
   inputJpgWithExif: getPath('Landscape_8.jpg'), // https://github.com/recurser/exif-orientation-examples/blob/master/Landscape_8.jpg
+  inputJpgWithIptcAndXmp: getPath('Landscape_9.jpg'), // https://unsplash.com/photos/RWAIyGmgHTQ
```

Comment:
Is there a smaller test image we could use? (Perhaps resize this one but maintain metadata?)

Reply:
Oops, sorry. I will definitely resize it.

Reply:
Done.